.php_cs.dist

```diff
@@ -6,18 +6,21 @@ $finder = PhpCsFixer\Finder::create()
 
 return PhpCsFixer\Config::create()
     ->setUsingCache(false)
-    ->setRules([
-        '@PSR2' => true,
-        '@Symfony' => true,
-        'no_useless_else' => true,
-        'no_useless_return' => true,
-        'ordered_class_elements' => true,
-        'ordered_imports' => true,
-        'phpdoc_order' => true,
-        'phpdoc_summary' => false,
-        'blank_line_after_opening_tag' => false,
-        'concat_space' => ['spacing' => 'one'],
-        'array_syntax' => ['syntax' => 'short'],
-        'yoda_style' => ['equal' => false, 'identical' => false, 'less_and_greater' => false],
-    ])
-    ->setFinder($finder);
\ No newline at end of file
+    ->setRules(
+        [
+            '@PSR2' => true,
+            '@Symfony' => true,
+            'no_useless_else' => true,
+            'no_useless_return' => true,
+            'ordered_class_elements' => true,
+            'ordered_imports' => true,
+            'phpdoc_order' => true,
+            'phpdoc_summary' => false,
+            'blank_line_after_opening_tag' => false,
+            'concat_space' => ['spacing' => 'one'],
+            'array_syntax' => ['syntax' => 'short'],
+            'declare_strict_types' => true,
+            'yoda_style' => ['equal' => false, 'identical' => false, 'less_and_greater' => false],
```

Comment:
I found the following rule to be useful:
`'binary_operator_spaces' => ['default' => null],`

Will prevent indentations to be dropped within array arguments e.g.
```
[
    'foo'    => 'bar'
    'bar'    => 'foo'
    'foobar' => true
]
```

Reply:
Could you please create an issue or PR in the default-plugin-repo for this suggestion, @dneustadt?
If this will be accepted by the majority we can apply this in a later PR.